modules/rosa-getting-started-revoking-admin-privileges-and-user-access.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-revoking-admin-privileges-and-user-access_{context}"]
+= Revoking administrator privileges and user access
+
+You can revoke `cluster-admin` or `dedicated-admin` privileges for a user as required.
+
+To revoke cluster access for a user, you must remove the user from your configured identity provider.
```

Comment:
Can this sentence be combined with the previous one? "You can revoke a users' `cluster-admin` or `dedicated-admin` privileges by removing them from your configured identity provider."

Reply:
These are two separate actions. The first sentence is about removing privileges from a user and the second sentence is about removing access to the cluster altogether from the user.

Reply:
@nalhadef after considering this further, I have made the first line in the section intro more explicit, to make it clearer that these are two separate topics. I have also added another line stating the following, to make the distinction between the two clear:

"You can follow the procedures in this section to revoke administrator privileges or revoke cluster access for a user."